nanozap/service/ChannelService.swift

```diff
@@ -31,33 +31,78 @@ class ChannelServiceMock : Channeler {
 }
 
 struct LndNode {
-    let pubKey : String
-    let alias : String
-    let color : String
-    let lastUpdate : Date
-    let totalCapacity : Int64
-    let numChannels : Int
+    let pubKey: String
+    let alias: String
+    let color: String
+    let lastUpdate: Date
+    let totalCapacity: Int64
+    let numChannels: Int
 }
 
 struct CloseChannelResult {
-    let txId : String
+    let txId: String
 }
 
-class ChannelService : Channeler {
+struct OpenChannelResult {
+    let response: Lnrpc_ChannelPoint
```

Comment:
Dont let the datatype from the transport escape the service into the UI. Translate them before returning.

The raw protobuf types have nothing to do in the UI